### DISCUSSION TITLE
what is Fl_Payment_Sum

### DISCUSSION THREAD

**Discussion by rioshaz** [YetiForceCompany/YetiForceCRM#14512] [How to] [answered]
![image](https://user-images.githubusercontent.com/54880467/113764420-8cbb9b00-9738-11eb-8836-fcd51d7ea3e0.png)

what does this file do and where doe sit mean,,, where does it get sis value from

**Comment by vovpff** [comment, marked as answer]
This is total sum of Payments In related to this Invoice

**Reply by rioshaz** [reply]
u mean total received against this invoice? because gross field is already a  total of invoice right ?

**Reply by vovpff** [reply]
Yes. There are can be 2 or more payments per one invoice.

**Reply by rioshaz** [reply]
u mean through payment in ?

**Reply by vovpff** [reply]
Yes

**Reply by rioshaz** [reply]
but if u go here  : https://gitdeveloper.yetiforce.com/index.php?module=FInvoice&view=Detail&record=791&parent=71&mid=129&mode=showDetailViewByMode&requestMode=full&tab_label=LBL_RECORD_DETAILS

the FL payment sum is : 
7 450.61

but there is no payment in against this invoice.. so i wonder...

**Reply by vovpff** [reply]
https://gitdeveloper.yetiforce.com/index.php?module=FInvoice&view=Detail&record=791&mode=showDetailViewByMode&requestMode=full&tab_label=LBL_RECORD_DETAILS

Pls cjeck here one more time this record and last actions in Updates tab. This mechanism works perfect.

When you explore system on devs site every tyme keep in your mind what gitdev database populated with random generated data.

**Reply by rioshaz** [reply]
cool man, thanks a lot buddy

**Reply by jmginfor** [reply]
Where can I find this FL_PAYMENT_SUM in the language translation system ? Because I want to use that in a generated PDF monthly statement  with $(custom : RecordsList)$ and show a shorter and more relevant title to this column.

**Reply by vovpff** [reply]
Add it into your language for Invoice module

**Reply by jmginfor** [reply]
@vovpff  When it's too simple...Thanks a lot
